llm_bench/python/who_what_benchmark/examples/openvino_cb.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import PosixPath
+import os
```

Comment:
Can you rename the file to `openvino_batched_eval`

Reply:
Done